code/test/util.js

```diff
@@ -257,7 +255,22 @@ const printInnerTrees = ({ Debugger, innerTrees }) => {
   }
 }
 
+// Create Accounts Alice Bob and Carol and fund them
+const createAccounts = async (initArgs) => {
```

Comment:
Create brand new accounts here? If the provider is instantiated with a private key, web3.eth.getAccounts() would only give you one account. Even if it instantiated by mnemonics (default setting), accounts at 0, 1, 2 often have balance and account 0 is used for execution by default.

Reply:
done 
Below was the alternate strategy which allowed each wallet to be assoicated with a resortAddress which could sign transactions. Useful if we want to test specific accounts
It was leaving account[0] as the "deployer" and having accounts 1 though 9 with 1000 native tokens

#### GANACHE TESTING

Note: The mnemonic and private keys shown are for testing only and should never be used in production. There is nothing special about this mnemonic you can generate your own using a tool such as https://iancoleman.io/bip39/ and then update your `.env` file with the private keys show when starting ganache.

For smart contract testing you should also install [ganache](https://trufflesuite.com/docs/ganache/quickstart.html). Once installed you should run ganache whenever using the `dev` or `ganache` networks. These networks have been configured to use port 7545 as such you can run ganache using the command 
```
ganache --port 7545 -m 'filter group there hunt fitness junior ghost park route jar entire clown allow rifle meadow'
```
by providing the mnemonic (using `-m`) you are guaranteed to receive the same mnemonic keys which you can configure in your `.env` file see `.env.sample` for an example.

Reply:
This has now been modified to create new accounts using a unique identifier
TT - Two Character Test Identifier (one per test file)
XX - Sub Test one per test 
99 - Wallet Identifier with test
Account creation logic
`   let alice = await CheckUtil.makeWallet('TT-ERC20-1', accounts[0], EFFECTIVE_TIME)`
Wallet Creation Function - note `effectiveTime` (the wallets effective creation time) is defined in the test and passed.
```
// makeWallet uses an index and unlocked web3.eth.account and creates and funds a ONEwallet
const makeWallet = async (salt, deployer, effectiveTime) => {
  const lastResortAccount = web3.eth.accounts.create()
  const { wallet, seed, hseed, root, client: { layers } } = await TestUtil.createWallet({
    salt: new BN(ONEUtil.keccak(salt)),
    effectiveTime: effectiveTime,
    duration: DURATION,
    maxOperationsPerInterval: SLOT_SIZE,
    lastResortAddress: lastResortAccount.address,
    spendingLimit: ONE_ETH
  })
  // Fund wallet
  await web3.eth.sendTransaction({
    from: deployer,
    to: wallet.address,
    value: HALF_ETH
  })
  const currentState = await getONEWalletState(wallet)
  return { wallet, seed, hseed, root, layers, lastResortAddress: lastResortAccount.address, currentState }
}
```